core/domain/topic_services.py

```diff
@@ -1730,8 +1730,16 @@ def get_chapter_counts_in_topic_summaries(
         overdue_chapters_count = 0
         total_chapter_counts = []
         published_chapter_counts = []
-        stories = [story_id_mapping[story_reference.story_id] for
-            story_reference in topic.canonical_story_references]
+        stories = []
+        for story_reference in topic.canonical_story_references:
+            if story_reference.story_id in story_id_mapping:
+                stories.append(story_id_mapping[story_reference.story_id])
+            else:
+                logging.error(
+            'Topic %s has canonical story references ' 
+            '%s but story reference %s is not a valid story.' % (topic.id
```

Comment:
Have a line break after '('. Put the comma directly after topic.id and leave a space after the comma.

Again, follow the style in the rest of the codebase (including for the closing paren). There are many examples to refer to. Please make sure you do this in all future PRs as well to reduce unnecessary back-and-forth.